### ISSUE TITLE
Consider 'claiming' a prometheus port allocation for stayrtr/rtrmon

### ISSUE THREAD

**Issue by ties** [bgp/stayrtr#10]
As a user [running stuff on metal/vms, not on k8s], to prevent the daemon overlapping with other ports, it would be helpful if stayrtr had an allocated port for it's prometheus endpoints.

To do so, reserve one in https://github.com/prometheus/prometheus/wiki/Default-port-allocations

**Comment by job** [comment]
Feel free to claim one, and submit a PR here

**Comment by randomthingsandstuff** [comment]
Yeah, what job said + if I don't get around to it before you do anyway. Laziness says "I coulda just done it while I wrote this comment, ah whatever" though.

**Comment by randomthingsandstuff** [comment]
9847 claimed in that wiki. Guess we just need to update things.
